src/spinner/_spinner.scss

```
@@ -1,45 +1,16 @@
 .spinner {
-  --spinner-background: 255, 255, 255;
-
-  position: relative;
-
-  width: 20px;
-  height: 20px;
-
-  margin: auto;
-
-  background: rgba(var(--spinner-background), 1);
-  background: linear-gradient(to right, rgba(var(--spinner-background), 1) 10%, rgba(var(--spinner-background), 0) 42%);
-
-  border-radius: 50%;
-
-  transform: translateZ(0);
-  animation: spinner 1.4s infinite linear;
-}
-
-.spinner__background {
-  position: absolute;
-  top: 0;
-  bottom: 0;
-  left: 0;
-  right: 0;
-
-  width: 75%;
-  height: 75%;
-
-  margin: auto;
-
-  border-radius: 50%;
-
-  content: "";
+  animation-name: spinner;
```

Let's make it shorthand

```suggestion
  animation: spinner 1.7s linear infinite;
```